tests/tests/test_fault_tolerance.py

```diff
@@ -110,8 +107,7 @@ def wait_for_download_retry_attempts(self, device, search_string):
 
         while int(time.time()) < timeout_time:
             output = device.run(
-                "journalctl -u %s -l --no-pager | egrep 'msg=\".*%s' | wc -l"
-                % (device.get_client_service_name(), search_string),
+                f"journalctl -u mender-updated -l --no-pager | egrep 'msg=\".*{search_string}' | wc -l",
```

Comment:
Nit: `s/egrep/grep -E`

Reply:
I have a pending PR with `grep -E`, long flags for journalctl, plus minor beautifications. Let me address everything there once the tests are _testing_